docs/source/conf.py

```diff
@@ -145,55 +136,47 @@ def get_python_version_from_pyproject():
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-# html_theme = "sphinx_rtd_theme"
-# html_theme = "bizstyle"
-html_theme = "alabaster"
+html_theme = "sphinx_book_theme"
 
 html_title = f"{project} v{version} Manual"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    "description": f"Simulation Tools for CTA v{version}",
```

Comment:
dont we need a description anymore?

Reply:
This option does not exist for the new theme (it appears on the top left below simtools; not really needed)